config/samples/watcher_v1beta1_watcher_tls_certificates.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: watcher-internal-svc
+spec:
+  dnsNames:
+  - watcher-internal.openstack.svc
+  duration: 43800h0m0s
+  issuerRef:
+    group: cert-manager.io
+    kind: Issuer
+    name: rootca-internal
+  secretName: cert-watcher-internal-svc
+  usages:
+  - key encipherment
+  - digital signature
+  - server auth
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: watcher-public-route
+spec:
+  dnsNames:
+  - watcher-public-watcher-openstack.apps-crc.testing
```

Comment:
I think this should be watcher-public-openstack.apps-crc.testing

Reply:
I think this still needs to be fixed.

Reply:
I fixed it wrong, sorry :), now it is fixed